docs/reference_guides/model_libraries/generic/unit_models/stream_scaler.rst

```diff
@@ -0,0 +1,46 @@
+Stream Scaler Block
+===================
+
+Stream Scaler Blocks are used to adjust size of streams to represent, for example, a stream being split across several identical units, which are then all modeled as a single IDAES unit
+
+Degrees of Freedom
+------------------
+
+Stream Scaler blocks generally have zero degrees of freedom.
+
+Model Structure
+---------------
+
+Stream Scaler Blocks consists of a single StateBlock (named properties), each with an inlet and outlet port.
```

Comment:
This is not correct from reading the code. The inlet state has a different name, and there is a separate (empty) outlet Block.